### ISSUE TITLE
Create stream screen

### ISSUE THREAD

**Issue by NilayMajorwar** [pclubiitk/Campus-Discuss-Frontend#17]
Create a component for the main screen of a stream. It should take as prop an array of posts:
```
[
  {
    id: 57,
    title: "Post 1",
    author: "John Doe",
    ... some other info ...
  },
  ... other posts ...
]
```

Then, it should render a list of `MinimizedPost` components, passing the post's title and author as props to each minimized post.

**Comment by ShivamMalhotra1994** [comment]
Diya Singhal and Shivam Malhotra are taking up this task.

**Comment by NilayMajorwar** [comment]
Closed by #19.
